webapp/src/components/modals/update_run_status_modal.tsx

```diff
@@ -133,33 +153,53 @@ const UpdateRunStatusModal = ({
         }
     };
 
+    const description = () => {
+        const broadcastChannelCount = run?.broadcast_channel_ids.length ?? 0;
```

Comment:
We also need to check here whether the setting is enabled or not with the new `status_update_broadcast_channels_enabled`. So maybe something like
```suggestion
        const broadcastChannelCount = 0;
        if (run?.status_update_broadcast_channels_enabled) {
            broadcastChannelCount = run?.broadcast_channel_ids.length ?? 0;
        }
```

Reply:
nice catch!